.github/release-drafter-config.yml

```diff
@@ -0,0 +1,21 @@
+# Configuration for Release Drafter: https://github.com/toolmantim/release-drafter
+template: |
+  Compatible with Elasticsearch (set version here).
+  $CHANGES
+
+# Setting the formatting and sorting for the release notes body
+name-template: Version (set version here)
+change-template: '* $TITLE (#$NUMBER)'
+sort-by: merged_at
+sort-direction: ascending
+
+# Organizing the tagged PRs into unified ODFE categories
+categories:
+  - title: "Enhancements"
+    labels: "enhancement"
+
+  - title: "Bug fixes"
+    labels: "bug fix"
```

Comment:
Is this a newly added label? I see two labels `bug` and `bug fix`.

Reply:
Yes, I added "bug fix" label in order to correctly categorize "Bug fixes" PRs in release notes.

Reply:
How is it different from the existing `bug` label? Why do we need a new label?

Reply:
The bug label is to categorize issues. But this 'bug fix' label is for PRs.

Reply:
Why the same label can't be used for both? IMO, it is very confusing and hard to remember which one should be used for issues and which one for PRs.

Reply:
"bug" is described as "Somthing isn't working" on the label itself. But "bug fix" in intuitively express that it is a kind of fixes, which it doesn't seem to be confusing at all. And typing in the "labels" search bar will auto-completes the labels, which we can easily choose 'bug' or 'bug fixes'. However, if you strongly insist that there's no need for a new label, I can change it.

Reply:
I don't think such distinction is necessary. `bug` on the PR can't mean that PR introduces a bug, it already means that PR fixes a bug. I also checked few other projects (guava and elasticsearch) and I don't think that they do such distinction. Not as part of this PR, but it will be good to adopt a better labelling scheme (for example similar to guava) and change `bug` to `defect`.

Reply:
That's fine. I've removed 'bug fix' label and updated the PR to use 'bug' label to recognize the 'Bug fixes' PRs.